### PR TITLE
modified a Dockerfile

### PR DESCRIPTION
**Pull request by RKemvou**
# WELCOME TO DEVOPS-EASY-LEARNING PULL REQUEST SYSTEM 
# Description of the change
modified a Dockerfile
> Detailed description here

## Checks and balances

- [ ] Tests added and run
- [ ] (External) Interfaces documented
- [ ] Has security implications (describe below)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
functionality to not work as expected)

## Related stories, issues and pulls

> Links to Pivotal Tracker stories, JIRA tickets or other pull requests here

## Security considerations

> Describe any security impact this change may have

### CODE REVIEW

**Review by fjen53 — APPROVED** [review]
well done

**Review by pdarso — APPROVED** [review]
looking good

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM tomcat:8.0-jre7
 COPY /home/ansible/webapp.war  /usr/local/tomcat/webapps
-VOLUME /volusagdqsgdqsgdqsme
+VOLUME /congo
```
Comment (francine-oss) [inline Dockerfile:3]: good

**Review by francine-oss — COMMENTED** [review]
good job

**Pull request closed without merging**